### PR TITLE
readme.md and build.txt update

### PR DESCRIPTION
**Pull request by DmitriiSukhomlinov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,31 +4,37 @@
 
 ### Prerequisites
 
-Make sure the Qt (>= 5.3.2) development libraries are installed:
+Make sure the Qt (>= 5.3.2 and < 5.15) development libraries are installed:
```
Comment (yalgaer) [inline README.md]: 5.4.2 <= ... <= 5.15?
I use 5.15 and 5.4.2 locally on Linux, so I know that it works for these OSes.

How do you know that UGENE builds with 5.3.2?

File: README.md [hunk 2]
```diff
@@ -4,31 +4,37 @@
...
 
+We highly recommend you to install Qt to any operating system with Qt online-installer. This is the main way for Windows and macOS operating systems, also available  for Linux. To install Qt, download the online-installer from Qt official site (http://www.qt.io/download/). Make sure, that you have installed the following components:
+* Qt Prebuild components
+* QtCharts
```
Comment (yalgaer) [inline README.md]: I think we do not use QtCharts anywhere in our code. Do we?

File: build.txt [hunk 3]
```diff
@@ -2,16 +2,23 @@
 Prerequisites
```
Comment (yalgaer) [inline build.txt]: Looks like this file contains duplicated content and should be dropped. 
If for some reason we can't remove it today we can put "read Readme.md" inside

**Review by yalgaer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DmitriiSukhomlinov) [reply README.md]: I've got an qt assert when I have tried to run UGENE with 5.15 on Windows. I will fix max version.
I do not know about 5.3.2, it just was here. I can change it to 5.4.2.

File: README.md (hunk 2 above)
Comment (DmitriiSukhomlinov) [reply README.md]: I've got it installed for some reason, but maybe, accidentally. Remove the line

File: build.txt (hunk 3 above)
Comment (DmitriiSukhomlinov) [reply build.txt]: I think we can remove this file at all

**Review by yalgaer — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (yalgaer) [reply README.md]: Just to confirm: we do not use/need this QtCharts.

**Pull request merged**